tests/ios/xcodeproj/Test-Imports-App-Project.xcodeproj/project.pbxproj

```diff
@@ -106,14 +104,6 @@
 			path = "test-imports-app";
 			sourceTree = "<group>";
 		};
-		49ACE10FF598298C0C4EA7D9 /* Resources2 */ = {
```

Comment:
the for the contribution! Is this intentional? We added this probably to test the situation when there are two image assets under one project...

Reply:
oh thanks for cathing this! you are right, this is a bug in my PR. `Resources2` are ignored because it also starts with `Resources`. Let me fix this bug

Reply:
Fixed